.github/workflows/publish-artifacts.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: publish artifacts
+# yamllint disable-line rule:truthy
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:
+      # Push events on default branch
+      - devel
+      # Push events to branches matching refs/heads/release-v*
+      - 'release-v*'
+jobs:
+  push:
+    name: Publish artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_IO_USERNAME }}
+          password: ${{ secrets.QUAY_IO_PASSWORD }}
+
+      - name: Set build environment based on Git branch name
+        if: github.ref == 'refs/heads/devel'
+        run: echo "BRANCH_NAME=devel" >> $GITHUB_ENV
+
+      - name: publish artifacts
+        # podman cannot pull images with both tag and digest
+        # https://github.com/containers/buildah/issues/1407
```

Comment:
@Madhu-1 do we have an issue here because of the limitation mentioned in the podman? I thought we only do `tag` based pulling. Isnt it the case?

Reply:
It was not working earlier this is taken from https://github.com/ceph/ceph-csi/blob/devel/.github/workflows/build-multi-stage.yaml#L15-L17. We do pull based on the `sha`, not the tag.

Reply:
sure, so we need only digest :) and not both ?  in short, do we need that caution/declaimer

Reply:
We need both. This is only to build the multistage container image.